e2e/backup/suite_test.go

```diff
@@ -164,7 +164,10 @@ var _ = BeforeSuite(func() {
 	SetupRancherResourceSet(o)
 
 	errC, ca := SetupOperator(testCtx, restConfig, operator.RunOptions{
-		ChartNamespace: ts.ChartNamespace,
+		ChartNamespace:       ts.ChartNamespace,
+		MetricsServerEnabled: true,
+		MetricsPort:          8080,
+		MetricsInterval:      5,
```

Comment:
should be `MetricsInterval:      1`  : speeds up the tests from ~1m -> ~15s. Might not seem important now but as tests are added this could add up